conf/route_control.py

```diff
@@ -782,35 +816,35 @@ def parse_args() -> Tuple[List[str], str, str]:
     if not args.i:
         parser.print_help()
         raise ValueError("interface must be specified")
-    return (args.i, args.ip, args.port)
+    return args.i, args.ip, args.port
 
 
-def register_signal_handlers(route_controller: RouteController) -> None:
+def register_signal_handlers(controller: RouteController) -> None:
     """Register signal handlers for SIGHUP, SIGINT, SIGTERM.
 
     Args:
         controller (RouteController): The route controller.
     """
     logger.info("Registering signals handlers.")
-    signal.signal(signal.SIGHUP, lambda number, _: route_controller.reconfigure(number))
-    signal.signal(signal.SIGINT, lambda number, _: route_controller.cleanup(number))
-    signal.signal(signal.SIGTERM, lambda number, _: route_controller.cleanup(number))
+    signal.signal(signal.SIGHUP, lambda number, _: controller.reconfigure(number))
+    signal.signal(signal.SIGINT, lambda number, _: controller.cleanup(number))
+    signal.signal(signal.SIGTERM, lambda number, _: controller.cleanup(number))
 
 
 if __name__ == "__main__":
     interface_arg, ip_arg, port_arg = parse_args()
     ipr = IPRoute()
-    ipdb = IPDB()
+    ndb = NDB(rtnl_debug=True)
```

Comment:
What's this `rtnl_debug` and do we really always want it set to True?

Reply:
If we set rtnl_debug=True all the loaded RTNL events could be  stored in the DB and those events may be exported later. For our case, we do no need it. Setting it to its default value False.